### PR TITLE
improved documentation for build_int_mul, build_int_sub, build_float_sub and build_float_mul

### PR DESCRIPTION
**Pull request by OranGot**
## Description

Improved documentations for functions in the title in builder.rs file with examples.

## How This Has Been Tested

I have not tested these changes since it's just a slight documentation change and I edited no code 

## Checklist

- [y ] I have read the [Contributing Guide](https://github.com/TheDan64/inkwell/blob/master/.github/CONTRIBUTING.md)
##Warning
This is my first ever pull request so it may not be the best.

### CODE REVIEW

File: src/builder.rs [hunk 1]
```diff
@@ -2410,7 +2410,36 @@ impl<'ctx> Builder<'ctx> {
 
         unsafe { Ok(T::new(value)) }
     }
-
+    ///creates integer subtraction given `lhs` and `rhs`. It returns either an `IntValue` or `BuilderError`
+    ///```rust,no_run
+    ///fn int_subtraction(lhs: i8, rhs: i8) -> i8{
```
Comment (TheDan64) [inline src/builder.rs:2415]: nit: can you please fix spacing after the slashes, and capitalize the C in creates

File: src/builder.rs [hunk 2]
```diff
@@ -2444,7 +2473,36 @@ impl<'ctx> Builder<'ctx> {
 
         unsafe { Ok(T::new(value)) }
     }
-
+    ///creates float subtraction given `lhs` and `rhs` as FloatValue. It returns either an `FloatValue` or `BuilderError`
+    ///```rust,no_run
+    ///fn float_subtraction(lhs: f16, rhs: f16) -> f16{
```
Comment (TheDan64) [inline src/builder.rs:2478]: Same

File: src/builder.rs [hunk 3]
```diff
@@ -2455,7 +2513,37 @@ impl<'ctx> Builder<'ctx> {
 
         unsafe { Ok(T::new(value)) }
     }
-
+    ///creates int  multiplication given `lhs` and `rhs` as IntValue. It returns either an
+    ///`IntValue` or `BuilderError`
+    ///```rust,no_run
+    ///fn int_mul(lhs: i8, rhs: i8) -> i8{
```
Comment (TheDan64) [inline src/builder.rs:2519]: Same

File: src/builder.rs [hunk 4]
```diff
@@ -2490,7 +2578,37 @@ impl<'ctx> Builder<'ctx> {
 
         unsafe { Ok(T::new(value)) }
     }
-
+    ///creates float multiplication
+    /// given `lhs` and `rhs` as FloatValue. It returns either an `FloatValue` or `BuilderError`
+    ///```rust,no_run
+    ///fn float_subtraction(lhs: f16, rhs: f16) -> f16{
```
Comment (TheDan64) [inline src/builder.rs:2584]: Same

**Review by TheDan64 — COMMENTED** [review]
Thank you! This looks pretty good, just needs some minor cleanup

File: src/builder.rs (hunk 1 above)
Comment (OranGot) [reply src/builder.rs]: done

**Comment by TheDan64** [comment]
@OranGot your PR will need some fixes before it can be merged

**Pull request closed without merging**